app/assets/javascripts/employer_profile.js

```diff
@@ -568,18 +568,29 @@ function checkPhone(textbox) {
   return true;
 }
 
-function checkZip(textbox) {
+function checkZip(textbox, child_index) {
   var phoneRegex = /^\d{5}$/;
   if (textbox.value == '') {
     textbox.setCustomValidity('Please fill out this zipcode field.');
   } else if(!phoneRegex.test(textbox.value)){
     textbox.setCustomValidity('please enter a valid zipcode.');
   } else {
     textbox.setCustomValidity('');
+    $.ajax({
+      type: 'get',
+      datatype: 'js',
+      url: '/employers/employer_profiles/get_counties',
+      data: {zip_code: textbox.value, child_index: child_index,},
+      success: function (response) {
+        $('#county' + child_index).show();
+        $('#county' + child_index).html(response);
+      }
+    });
```

Comment:
If I mistype a zip code I can get into a state where the zip box is gone and I have to reload the page